requirements/edx/base.txt

```diff
@@ -734,7 +734,7 @@ openedx-events==0.8.2
     # via -r requirements/edx/base.in
 openedx-filters==0.6.1
     # via -r requirements/edx/base.in
-ora2==4.2.0
+ora2==4.3.0
```

Comment:
Can you update base.in to use `ora2>=4.3.0`? If an older version of ORA were used for some reason, it would break after this change to CourseWaffleFlag.